monitoring/mongodb/alerts.yaml

```diff
@@ -139,3 +139,45 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      avg_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1h]) == 3
```

Comment:
what is happening when recovering fails: does the pod restart, or does it stay in recovering forever?

if the pod restarts, then the average may well be below 3...

Reply:
Indeed we have the livenessProbe configured to force the pod restart otherwise it would stay in the recovering state forever

Reply:
After checking with @williamlardier  the node actually stays stuck as it's responding to the probe , so it is not restarted so the average will be 3

Reply:
(when the recovering fails, this is another issue IMHO, in general a recovering instance might run for hours without issue; when the recovering fails, we would have other alerts anyway, to notify that not all mongo are up)